roles/confluent.common/defaults/main.yml

```diff
@@ -1,4 +1,13 @@
 ---
+### Configures package repositories on hosts. By default will configure confluent's deb/yum repositories. Possible options: none, confluent, custom. Must also set custom_yum_repofile_filepath or custom_apt_repo_filepath if using custom
+repository_configuration: confluent
```

Comment:
wonder if theres a way to make this var backwards compatible to the original vars